### PR TITLE
Google OAuth2 Example needs 2 more configuration

### PR DESCRIPTION
**Pull request by LobeTia**
Added callbackUrl & scope into README and config/passport.js

### CODE REVIEW

File: lib/passports.js [hunk 1]
```diff
@@ -101,14 +101,26 @@ module.exports = {
...
-              app.services.FootprintService.update('Passport', passport.id, passport) .then(passport => {
-                app.services.FootprintService.find('user', passport[0].user.id)
-                  .then(user => {
-                    next(null, user)
-                  })
-                  .catch(next)
-              }).catch(next)
+             // TODO: Must be tested for check if ok for both Sequelize and Waterline
+             // Save any updates to the Passport before moving on
+              app.services.FootprintService.find('Passport', passport.id)
```
Comment (jaumard) [inline lib/passports.js]: Why searching for a new passport instance  when you already have it ? The find is few lines above so if you have it here no need to do another search just if(passport) should be enough

File: lib/passports.js [hunk 2]
```diff
@@ -101,14 +101,26 @@ module.exports = {
...
+                .then(passport => {
+                  // I can't rely on the update results because on trailpack-sequelize return an array with the updated rows counter
+                  // So, first I read passport model
+                  app.services.FootprintService.update("Passport", passport.id, passport)
+                    .then(() => {
+                      // I read UserId field (very bad) 
+                      return app.services.FootprintService.find('user', passport.UserId)
```
Comment (jaumard) [inline lib/passports.js]: `passport.UserId` will not work for waterline and sequelize, don't remember witch one is failing on this

File: lib/passports.js (hunk 1 above)
Comment (LobeTia) [inline lib/passports.js]: Sorry, I created the pull request and committed other 2 changes on master
Only a04e06e & 78a6eec are related to this pull request
The other 2 changes is a WIP related to Sequelize (actually this trailpack doesn't work on Sequelize due to some FootprintService differences compared to Waterline)
Refuses without problems this PR, in these days I will create the different branches with both the fix and create other 2 PRs

Comment (jaumard) [inline lib/passports.js]: Ho ok ^^ :) I use it with sequelize only (I don't use waterline) with no problem but never use third party login like google auth/facebook/twitter... so yes maybe some problem may occur on this part as I don't know how unit test those... 
If you need help/feedbacks/advices, don't hesitate to reach me on gitter ;) 
I'll refuse this one and wait for the others then thanks !

**Pull request closed without merging**